application/buildSrc/src/main/java/com/stepango/forma/feature/AndroidLibrary.kt

```diff
@@ -15,7 +15,8 @@ data class AndroidLibraryFeatureConfiguration(
     val consumerMinificationFiles: Set<String> = emptySet(),
     val manifestPlaceholders: Map<String, Any> = emptyMap(),
     val selfValidator: Validator = validator(LibraryModule),
-    val dataBinding: Boolean = false
+    val dataBinding: Boolean = false,
```

Comment:
Hi, @atetc nice to see you here! 
dataBindings removed from androidLibrary implementation it is a different module type now, viewBinding should not be allowed in androidLibrary as well. please reference dataBindings module type implementation for more details